parm/use_cases/met_tool_wrapper/ASCII2NC/ASCII2NC_python_embedding.conf

```
@@ -42,7 +42,7 @@ LEAD_SEQ = 0
 LOOP_ORDER = times
 
 # Verbosity of MET output - overrides LOG_VERBOSITY for ASCII2NC only
-LOG_ASCII2NC_VERBOSITY = 1
+#LOG_ASCII2NC_VERBOSITY = 1
```

```suggestion
#LOG_ASCII2NC_VERBOSITY = 2
```
shouldn't this be 2?